pkg/action/build-uki.go

```diff
@@ -632,9 +632,9 @@ func (b *BuildUKIAction) imageFiles(sourceDir string) (map[string][]string, erro
 		"loader/entries": {},
 		"loader/keys":    {},
 		"loader/keys/auto": {
-			filepath.Join(b.keysDirectory, "PK.der"),
-			filepath.Join(b.keysDirectory, "KEK.der"),
-			filepath.Join(b.keysDirectory, "db.der"),
+			// filepath.Join(b.keysDirectory, "PK.der"),
+			// filepath.Join(b.keysDirectory, "KEK.der"),
+			// filepath.Join(b.keysDirectory, "db.der"),
```

Comment:
Same as comment above. Maybe I should undo this change to limit this PR to the relevant stuff?

Reply:
I think, that if you do manual importing on the firmware, it accepts the der files IIRC? at least on qemu you have to select those files

Reply:
maybe it's best to have all of them if it doesn't hurt. might be handy to have them in the installer iso in any case

Reply:
I'm not sure appending them one after the other is correct: https://stackoverflow.com/a/21185723
Also, `openssl` only decodes the first one found when multiple are in a file.

Reply:
There is a function that parses multiple keys: https://github.com/kairos-io/enki/pull/86/files#diff-4c9460930d2508990a466375e10496dbb61115e44bcc6ffb733b0487d7052ba5R200

so this should be ok to do.